pkg/operator/ceph/object/notification/controller_test.go

```diff
@@ -54,19 +54,25 @@ var (
 
 var createWasInvoked bool
 
-func mockSetup() {
+func mockSetup() func() {
 	createWasInvoked = false
-	Create = func(p Provisioner, bucket *bktv1alpha1.ObjectBucket, topicARN string, notification *cephv1.CephBucketNotification) error {
+	createNotificationFunc = func(p provisioner, bucket *bktv1alpha1.ObjectBucket, topicARN string, notification *cephv1.CephBucketNotification) error {
 		createWasInvoked = true
 		return nil
 	}
-	DeleteAll = func(p Provisioner, bucket *bktv1alpha1.ObjectBucket) error {
+	deleteAllNotificationsFunc = func(p provisioner, bucket *bktv1alpha1.ObjectBucket) error {
 		return nil
 	}
+	return func() {
+		// returned function is doing the cleanup
```

Comment:
Can we move this function to just above `mockSetup`? This way, it shows up when I hover over the function in VSCode to understand the behavior.